### PR TITLE
Added aria labels to every date button

### PR DESCRIPTION
**Pull request by elliotmendiola**
## PR Type
What kind of change does this PR introduce?
Accessibility improvements

```
[x] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Demos changes
[ ] Other... Please describe:
```

## What is the current behavior?
There are no aria labels at all so nothing is accessible.

## What is the new behavior?
There are aria labels that use the date pipe so they are automatically localized if you have locales set up properly

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: dev/src/ion2-calendar/components/calendar.component.ts [hunk 1]
```diff
@@ -29,16 +29,18 @@ export const ION_CAL_VALUE_ACCESSOR: Provider = {
     <div class="title">
       <ng-template [ngIf]="_showMonthPicker" [ngIfElse]="title">
         <ion-button type="button"
-                fill="clear"
-                class="switch-btn"
-                (click)="switchView()">
+                    fill="clear"
+                    class="switch-btn"
+                    [attr.aria-lable]="getDate(monthOpt.original.time) | date:MONTH_DATE_FORMAT"
```
Comment (ulesta) [inline dev/src/ion2-calendar/components/calendar.component.ts]: typo in `aria-label`

File: dev/src/ion2-calendar/components/calendar.component.ts [hunk 2]
```diff
@@ -29,16 +29,18 @@ export const ION_CAL_VALUE_ACCESSOR: Provider = {
...
+                    (click)="switchView()">
           {{ _monthFormat(monthOpt.original.time) }}
           <ion-icon class="arrow-dropdown"
                     [name]="_view === 'days' ? 'md-arrow-dropdown' : 'md-arrow-dropup'"></ion-icon>
         </ion-button>
       </ng-template>
       <ng-template #title>
-        <div class="switch-btn">
+        <div class="switch-btn"
+             [attr.aria-lable]="getDate(monthOpt.original.time) | date:MONTH_DATE_FORMAT">
```
Comment (ulesta) [inline dev/src/ion2-calendar/components/calendar.component.ts]: typo in aria-label

Comment (elliotmendiola) [reply dev/src/ion2-calendar/components/calendar.component.ts]: 🤦‍♂️

**Comment by elliotmendiola** [comment]
Fixed typo

**Pull request merged**